app/Policies/MessagePolicy.php

```diff
@@ -46,4 +48,25 @@ public function forceDelete(User $user, Message $message): bool
     {
         return false;
     }
+
+    public function sendToRecipient(User $user, User $targetUser): bool
+    {
+        $isUserDeveloper = $user->hasAnyRole([
+            Role::DEVELOPER,
+            Role::DEVELOPER_JUNIOR,
+            Role::DEVELOPER_STAFF,
+        ])
+            || $user->getAttribute('Permissions') >= Permissions::JuniorDeveloper;
+
+        /**
+         * TODO check user privacy settings
+         */
```

Comment:
Are there other user privacy settings I'm not aware of, or is this in reference to some future feature?

Reply:
It's a reference to a future feature; I found numerous references to the feature elsewhere in the codebase (search for "check user privacy settings"). I assume this data will be held in `UserAccounts.preferences`.